plugins/orchestrator/src/components/next/utils.ts

```diff
@@ -1,7 +1,11 @@
 import { ProcessInstanceState } from '@janus-idp/backstage-plugin-orchestrator-common';
 
-export const firstLetterCapital = (text: string) =>
-  text && text[0].toUpperCase() + text.slice(1).toLowerCase();
+export const firstLetterCapital = (text: string | undefined) => {
```

Comment:
IMHO having these `utils.ts` modules all over the place without a clear context makes it harder to reuse code later and discover where similar functionality should be placed. 
For example in this case, this function is something I would love to have as part of the `String` object API but, since I don't have it, I would create first a dir called `src/utils/`. Inside, I would create a `StringUtils.ts` module and move this functionality there. If tomorrow I'll write a `toSnakeCase` function I'll know that the best place for it would be inside the `StringUtils.ts` module. WDYT?

Reply:
Agree, I will move it